src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -50,18 +50,33 @@ def get(self, request, project):
         :pparam string project_slug: the slug of the project.
         :auth: required
         """
-        queryset = UserReport.objects.filter(
-            project=project,
-            group__isnull=False,
-        ).select_related('group')
-
-        status = request.GET.get('status', 'unresolved')
-        if status == 'unresolved':
-            queryset = queryset.filter(
-                group__status=GroupStatus.UNRESOLVED,
+        try:
+            environment = self._get_environment_from_request(
+                request,
+                project.organization_id,
             )
-        elif status:
-            return Response({'status': 'Invalid status choice'}, status=400)
+        except Environment.DoesNotExist:
+            queryset = Environment.objects.none()
+        else:
+            if environment is not None:
+                queryset = UserReport.objects.filter(
```

Comment:
You can rewrite these as well similar to my other comment about chaining `filter()`s.

Reply:
@mattrobenolt quick question. Is there any noticeable performance difference I should be aware of when chaining a filter versus doing it all at once?

Reply:
It technically duplicates the QuerySet object for each `.filter()` as it's chained since the objects are immutable. But as far as the resulting query, it ends up doign the same thing, and the performance here of copying objects isn't negligible.

If it were significant and we cared, you could construct kwargs like:

```python
filters = {
    'project': project,
    'group__isnull': False,
 }
if environment is not None:
    filters['environment'] = environment

queryset = UserReport.objects.filter(**filters)
```